shippable.yaml

```diff
@@ -0,0 +1,39 @@
+---
+language: python
+
+python:
+  - 3.6
+
+
+branches:
+  only:
+    - unreviewed
+
+build:
+  # pre_ci_boot:
+  #   image_name: 659542948624.dkr.ecr.us-east-1.amazonaws.com/python-lambda-shippable-image
```

Comment:
This exposes our prod aws account number, which I guess is fine, but just worth thinking about

Reply:
So in theory it doesn't matter - these account numbers are expected to be published and often are - we need to put this image up.   However later we'll shuffle around and probably have the account dedicated for any public images which will make things easier anyway. 

@paddledave can you please approve if there's nothing else.